lib/Settings/Admin.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ * Nextcloud - passman
+ *
+ * @copyright Copyright (c) 2016, Sander Brand (brantje@gmail.com)
+ * @copyright Copyright (c) 2016, Marcos Zuriaga Miguel (wolfi@wolfi.es)
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\Passman\Settings;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\GuzzleException;
+use OCP\App\IAppManager;
+use OCP\AppFramework\Http\TemplateResponse;
+use OCP\IConfig;
+use OCP\IL10N;
+use OCP\Settings\ISettings;
+
+class Admin implements ISettings {
+
+	protected IConfig $config;
+	private IL10N $l;
+	private IAppManager $appManager;
+
+	/**
+	 * Admin constructor.
+	 * @param IConfig $config
+	 * @param IL10N $l
+	 * @param IAppManager $appManager
+	 */
+	public function __construct(IConfig $config, IL10N $l, IAppManager $appManager) {
+		$this->config = $config;
+		$this->l = $l;
```

Comment:
Can we have a more meaningful property and variable naming?

Reply:
if you mean $l, the term is currently used that way throughout the project. just wanted it to be the same.
how would you like it?

Reply:
Oh, if that is the case, let's leave it short, but short variable/property names are... evil